pkgs/development/python-modules/pyflipper/default.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pyserial,
+  websocket-client
+}:
+
+buildPythonPackage rec {
+  pname = "pyflipper";
+  version = "unstable-2024-04-15";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```

and add `build-system = [ setuptools ];`